### PR TITLE
3x tweaks

### PR DESCRIPTION
**Pull request by manolama**

### CODE REVIEW

File: core/src/main/java/net/opentsdb/query/filter/TagValueLiteralOrFactory.java [hunk 1]
```diff
@@ -44,11 +43,23 @@ public QueryFilter parse(final TSDB tsdb,
     if (node == null) {
       throw new IllegalArgumentException("Node cannot be null.");
     }
-    try {
-      return (QueryFilter) mapper.treeToValue(node, TagValueLiteralOrFilter.class);
-    } catch (JsonProcessingException e) {
-      throw new IllegalArgumentException("Failed to parse TagValueLiteralOr", e);
+    TagValueLiteralOrFilter.Builder builder = TagValueLiteralOrFilter.newBuilder();
+    JsonNode n = node.get("tagKey");
```
Comment (smrutilal2) [inline core/src/main/java/net/opentsdb/query/filter/TagValueLiteralOrFactory.java]: shouldn't this be `node.get("key")`

File: core/src/main/java/net/opentsdb/query/filter/TagValueLiteralOrFilter.java [hunk 2]
```diff
@@ -179,12 +179,12 @@ public static Builder newBuilder() {
   @JsonIgnoreProperties(ignoreUnknown = true)
   public static class Builder {
     @JsonProperty
-    private String tagKey;
+    private String key;
```
Comment (smrutilal2) [inline core/src/main/java/net/opentsdb/query/filter/TagValueLiteralOrFilter.java]: I like this change.

File: core/src/main/java/net/opentsdb/query/filter/TagValueRangeFilterFactory.java [hunk 3]
```diff
@@ -49,11 +48,23 @@ public QueryFilter parse(final TSDB tsdb,
     if (node == null) {
       throw new IllegalArgumentException("Node cannot be null.");
     }
-    try {
-      return (QueryFilter) mapper.treeToValue(node, TagValueRangeFilter.class);
-    } catch (JsonProcessingException e) {
-      throw new IllegalArgumentException("Failed to parse TagValueRegexFilter", e);
+    TagValueRangeFilter.Builder builder = TagValueRangeFilter.newBuilder();
+    JsonNode n = node.get("tagKey");
```
Comment (smrutilal2) [inline core/src/main/java/net/opentsdb/query/filter/TagValueRangeFilterFactory.java]: shouldn't this be node.get("key")

File: core/src/main/java/net/opentsdb/query/filter/TagValueLiteralOrFactory.java (hunk 1 above)
Comment (smrutilal2) [reply core/src/main/java/net/opentsdb/query/filter/TagValueLiteralOrFactory.java]: Is this for backward compatibility?

File: core/src/main/java/net/opentsdb/query/filter/TagValueRangeFilterFactory.java (hunk 3 above)
Comment (manolama) [reply core/src/main/java/net/opentsdb/query/filter/TagValueRangeFilterFactory.java]: backwards compatibility. See three lines below.

File: core/src/main/java/net/opentsdb/query/filter/TagValueLiteralOrFactory.java (hunk 1 above)
Comment (manolama) [reply core/src/main/java/net/opentsdb/query/filter/TagValueLiteralOrFactory.java]: yup

**Pull request merged**